server/methods/accounts/accounts.app-test.js

```diff
@@ -66,7 +66,7 @@ describe("Account Meteor method ", function () {
       Accounts.remove({});
     });
 
-    it("should allow user to add new addresses", function (done) {
+    it("should allow user to add new addresses", function () {
```

Comment:
I removed all of the `done` callbacks based on @zenweasel's comment (none of these tests are async)